src/c++/perf_analyzer/inference_profiler.h

```diff
@@ -463,6 +483,14 @@ class InferenceProfiler {
   bool include_lib_stats_;
   bool include_server_stats_;
   std::shared_ptr<MPIDriver> mpi_driver_;
-};
 
+#ifndef DOCTEST_CONFIG_DISABLE
+  friend TestInferenceProfiler;
+  InferenceProfiler(LoadParams& lp)
```

Comment:
Can this be changed to do the no-parameter version?

Reason is because all future unit tests that utilize this won't necessarily need to supply this data member. With a default constructor, this data member can still be set with the dot accessor inside the unit test.

Reply:
I dont want to do that in this ticket.  
If the need arises, lets update it at that point.

Reply:
updated